.gitignore

```diff
@@ -10,3 +10,4 @@ service-account.json
 *.csv
 *.txt
 名称未設定.*
```

Comment:
remove? :-)

Reply:
I wanna keep this as close as the original, so that we can eventually merge back in.